desktop/core/src/desktop/appmanager.py

```diff
@@ -261,6 +261,9 @@ def load_apps(app_blacklist):
   DESKTOP_APPS = []
 
   for sdk_app in pkg_resources.iter_entry_points("desktop.sdk.application"):
+    
+    if desktop.conf.CSP_NONCE.get():
+      app_blacklist.extend(['oozie', 'jobbrowser', 'rdbms', 'search', 'zookeeper'])
```

Comment:
Suggestion: Can you add a comment why we are extending the blacklist here for specific apps?